### PR TITLE
[IRRC-60] Create Screening Table view with search filters

### PR DESCRIPTION
**Pull request by dzldebby**

### CODE REVIEW

File: frontend/src/pages/Screening.vue
Comment (thchia) [inline frontend/src/pages/Screening.vue]: Just start from `/api/...`

Comment (thchia) [inline frontend/src/pages/Screening.vue]: Needs the relevant PR from @lauratsui merged before this will work.

**Review by thchia — CHANGES_REQUESTED** [review]
See the comment on the API URL.

File: frontend/src/pages/Screening.vue
Comment (dzldebby) [reply frontend/src/pages/Screening.vue]: @thchia will wait for #39 to be merged

Comment (dzldebby) [reply frontend/src/pages/Screening.vue]: Edited

**Comment by dzldebby** [comment]
Marking as WIP as waiting for #39 to merged in

File: frontend/src/pages/Screening.vue
Comment (thchia) [reply frontend/src/pages/Screening.vue]:
```suggestion
const API_URL = "/api/students/?status=Screening";
```
Is actually what I meant.

File: frontend/src/pages/Screening.vue
Comment (thchia) [inline frontend/src/pages/Screening.vue]: Remember, we should not make any assumptions about what the ID is. When we deploy the final version of the database for real use, there is no guarantee that `"Bangla"` will have an ID of `1`. It would be better if the backend helps us out here by returning the JOINed data (like what we discussed during the planning), so that we get the description back as part of `this.data`.

**Review by thchia — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Screening.vue
Comment (dzldebby) [reply frontend/src/pages/Screening.vue]: Yeah, hard-coded just to see it all come together on the front-end. The backend PR that addresses this is #42 right, will review it once the conflicts in #42 are resolved

Comment (dzldebby) [reply frontend/src/pages/Screening.vue]: Update: Awaiting modification to include native language text when requesting for list of students

**Review by nondescryptid — CHANGES_REQUESTED** [review]
Looks good, just 2 comments:
1. Dropdown: "Limited" should be "Simple" instead (in 20201009200757_students.js, English Proficiency is an enum with "No", "Little", "Simple", "Intermediate") 
2. This doesn't work on its own as the GET method in #45 has not been incorporated in this PR, though the frontend code is configured to use that method. But after I merged in the latest master, it works great! Otherwise, it says it cant read NativeLanguage and that column goes missing.

**Comment by dzldebby** [comment]
> Looks good, just 2 comments:
> 
> 1. Dropdown: "Limited" should be "Simple" instead (in 20201009200757_students.js, English Proficiency is an enum with "No", "Little", "Simple", "Intermediate")
> 2. This doesn't work on its own as the GET method in #45 has not been incorporated in this PR, though the frontend code is configured to use that method. But after I merged in the latest master, it works great! Otherwise, it says it cant read NativeLanguage and that column goes missing.

Revised to 'Simple', and merged in new GET method. Thanks Tomoe!

**Review by nondescryptid — APPROVED** [review]
LGTM! Works perfectly now

**Review by thchia — APPROVED** [review]
Since my previous comments are addressed and it has been tested locally by @szktm, I'm good with this.

**Pull request merged**
